src/styles/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  margin: 0;
+  display: flex;
+  font-family: Roboto, serif;
+  font-weight: 400;
+  text-align: left;
+  flex-direction: column;
+  justify-content: center;
+  padding: 32px 16px 16px;
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px #f3f3f3 solid;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    padding-bottom: 40px;
+    align-self: center;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0;
+    padding-bottom: 4px;
+    color: $main-color;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    margin: 0;
+    padding-bottom: 16px;
+    color: $second-color;
+  }
+
+  &__reviews {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: 24px;
+
+    &-amount {
+      margin: 0;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      text-align: right;
+      color: $main-color;
+      display: flex;
+    }
+  }
+
+  &__price {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: 16px;
+
+    &-label {
+      margin: 0;
+      text-align: left;
+      font-size: 12px;
+      line-height: 18px;
+      color: $second-color;
+    }
+
+    &-amount {
+      margin: 0;
+      font-size: 16px;
+      line-height: 18px;
+      text-align: right;
+      font-weight: 700;
+      color: $main-color;
+    }
+  }
+
+  &__button {
+    box-sizing: border-box;
+    border-radius: 5px;
+    background-color: $blue;
```

Comment:
The variable `$blue` is used here, but it's not defined in this file. Ensure that it is defined in a file that is imported into this one, or define it here.